test/extensions/filters/http/aws_lambda/aws_lambda_filter_test.cc

```diff
@@ -69,6 +69,24 @@ TEST_F(AwsLambdaFilterTest, DecodingHeaderStopIteration) {
   EXPECT_EQ(Http::FilterHeadersStatus::StopIteration, result);
 }
 
+TEST_F(AwsLambdaFilterTest, HostHeaderLambdaTrue) {
+  setupFilter({arn_, InvocationMode::Synchronous, true /*passthrough*/});
+  if  Runtime
+      ::maybeSetRuntimeGuard("envoy.reloadable_features.lambda_sanitize_host_headers",true) {
+    Http::TestRequestHeaderMapImpl headers;
+    EXPECT_STREQ(headers.getHostValue(), "lambda");
+  }
+}
+
+TEST_F(AwsLambdaFilterTest, HostHeaderLambdaFalse) {
+  setupFilter({arn_, InvocationMode::Synchronous, true /*passthrough*/});
+  if Runtime
+    ::maybeSetRuntimeGuard("envoy.reloadable_features.lambda_sanitize_host_header",false) {
```

Comment:
Do you want to try setting the runtime guard in the tests like shows here? in case these would not run into the same format issue.

https://github.com/envoyproxy/envoy/blob/65273b2a9b25e9650a4379c9eeb0c1c48f29089c/test/extensions/filters/http/oauth2/filter_test.cc#L420-L423

Reply:
Meanwhile, I have not run this recently myself. So the exact step might have changed.

There is an option to run some tools within Envoy repo that formats the code. I think one of these steps should be working.

```
./ci/do_ci.sh format
export CLANG_FORMAT=/opt/llvm/bin/clang-format; ./tools/code_format/check_format.py fix;
./tools/local_fix_format.sh

```